### PR TITLE
Update  git / workflow / logging

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
Update Link Name

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/git/workflow/logging/limiting-log-output-by-time.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/git/workflow/logging/limiting-log-output-by-time.md?ref=960f6b92aed1c1cf50703334c07a3138df99dc53)

File: git/workflow/logging/limiting-log-output-by-time.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ aspects:
 
 links:
 
-  - '[git-scm.com](http://git-scm.com/book/en/v2/Git-Basics-Viewing-the-Commit-History){website}'
+  - '[Limiting Log Output Official Documentation](http://git-scm.com/book/en/v2/Git-Basics-Viewing-the-Commit-History){website}'
```
Comment (kapnobatai137) [inline git/workflow/logging/limiting-log-output-by-time.md]:
```suggestion
  - '[Limiting Log Output](http://git-scm.com/book/en/v2/Git-Basics-Viewing-the-Commit-History){documentation}'
```

File: git/workflow/logging/make-your-log-output-pretty.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ aspects:
 
 links:
 
-  - '[git-scm.com](http://git-scm.com/book/en/v2/Git-Basics-Viewing-the-Commit-History){website}'
+  - '[Git Basics Official Documentation](http://git-scm.com/book/en/v2/Git-Basics-Viewing-the-Commit-History){website}'
```
Comment (kapnobatai137) [inline git/workflow/logging/make-your-log-output-pretty.md]:
```suggestion
  - '[Git Basics](http://git-scm.com/book/en/v2/Git-Basics-Viewing-the-Commit-History){documentation}'
```

File: git/workflow/logging/view-your-commit-history-in-a-graph.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ aspects:
 
 links:
 
-  - '[git-scm.com](https://git-scm.com/book/en/v2/Git-Basics-Viewing-the-Commit-History){website}'
+  - '[Git Basics Official Documentation](https://git-scm.com/book/en/v2/Git-Basics-Viewing-the-Commit-History){website}'
```
Comment (kapnobatai137) [inline git/workflow/logging/view-your-commit-history-in-a-graph.md]:
```suggestion
  - '[Git Basics](http://git-scm.com/book/en/v2/Git-Basics-Viewing-the-Commit-History){documentation}'
```

**Pull request merged**